### ISSUE TITLE
BertForPreTraining predicts the masked token with ZERO accuracy.

### ISSUE THREAD

**Issue by lazyfuzzypringle** [google-research/bert#666]
Hi, I run the pretrained model BertForPreTraining and test it on my own text data. Because BERT has knowledge about language so I expect it to be able to predict the masked tokens with a reasonable accuracy, however it is zero. I run the model with model.eval() mode. Does it make sense that it predicts the masked tokens with zero?

Thanks for the help.
